src/dswx_sar/region_growing.py

```diff
@@ -0,0 +1,329 @@
+import os
+import logging
+import time
+import mimetypes
```

Comment:
```suggestion
import mimetypes
import logging
import os
import time
```
nit - PEP8 import order